reloadconf/__init__.py

```diff
@@ -191,6 +264,13 @@ def poll(self):
                 # If command is not running and config is valid, start command.
                 self.start_command()
 
+    def kill(self):
+        """Kill the running command."""
+        if self.process is not None:
+            LOGGER.info('Killing command...')
+            self.process.kill()
```

Comment:
I think you should `self.process.wait()` here. I suspect (but I'm not 100% sure) that if the child process sticks around the parent (this one) may end up being a zombie. An even more fine control would involve SIGTERM -> wait() -> SIGKILL -> wait() -> giveup as in:
http://psutil.readthedocs.io/en/latest/#terminate-my-children
...but that is probably too much.